bounce/server/api/clubs.py

```diff
@@ -34,28 +35,38 @@ class ClubEndpoint(Endpoint):
 
     @validate(PutClubRequest, GetClubResponse)
     async def put(self, request, name):
-        """Handles a PUT /clubs/<name> request by updating the club with
+        """Handles a PUT /clubs/<name>?access=<role> request by updating the club with
```

Comment:
We can't accept the role as part of the request, otherwise any attacker would just be able to put the Admin role in the request and do anything they like. We'll need to use `@verify_token` to check that the user is signed in, and then use the user ID in the token to look up the user's role in the DB before we before the action they requested.

Reply:
Right, that makes sense. I'll try to implement the changes.